build.gradle.kts

```diff
@@ -35,21 +33,3 @@ val dokkaPublish by tasks.registering {
         finalizedBy(tasks.named("closeAndReleaseSonatypeStagingRepository"))
     }
 }
-
-apiValidation {
-    // note that subprojects are ignored by their name, not their path https://github.com/Kotlin/binary-compatibility-validator/issues/16
-    ignoredProjects += setOf(
-        // NAME                    PATH
-        "search-component",    // :plugins:search-component
-        "frontend",            // :plugins:base:frontend
-
-        "kotlin-analysis",     // :kotlin-analysis
-        "compiler-dependency", // :kotlin-analysis:compiler-dependency
-        "intellij-dependency", // :kotlin-analysis:intellij-dependency
-
-        "integration-tests",   // :integration-tests
-        "gradle",              // :integration-tests:gradle
-        "cli",                 // :integration-tests:cli
-        "maven",               // integration-tests:maven
-    )
-}
```

Comment:
bcv-mu is applied on individual subprojects, so this configuration can be removed.